package.json

```diff
@@ -37,9 +37,9 @@
     "prepare": "mkdir -p public/js",
     "bootstrap": "cp -r node_modules/bootstrap/dist/* public/",
     "jquery": "cp -r node_modules/jquery/dist/* public/js/",
-    "forge": "cd node_modules/node-forge && npm install && npm run bundle && cp js/forge.bundle.js ../../public",
+    "forge": "cd node_modules/node-forge && sudo npm install && sudo npm run bundle && sudo cp js/forge.bundle.js ../../public",
     "browser": "browserify -g brfs samling.js | sed \"s/require[(]'node-forge'[)]/window.forge/g\" > public/bundle.js",
-    "build": "npm run prepare && npm run bootstrap && npm run jquery && npm run forge && npm run browser",
+    "build": "npm run prepare && npm run bootstrap && npm run jquery && npm run forge && sudo npm run browser",
```

Comment:
do these changes prevent running these commands after it has been deployed? if that is even necessary?